front/app/utils/errorUtils.ts

```diff
@@ -173,3 +175,19 @@ export const handleHookFormSubmissionError = (
     });
   }
 };
+
+export const handleBlockedUserError = (status, data) => {
```

Comment:
Can you add types here for `status` and `data`? We're working towards enabled strict mode, which requires that.

Reply:
data can be anything, its ok to use `any` type here?